db/db.go

```diff
@@ -0,0 +1,23 @@
+package db
+
+import (
+	"gopkg.in/mgo.v2"
+	"log"
+)
+
+type TripDao struct {
+	Server   string
+	Database string
+}
+
+var MgoSession *mgo.Session
+
+func init() {
+	session, err := mgo.Dial("mongo:27017")
```

Comment:
This should be configurable.

Reply:
added environment variables MONGO_HOST for IP address, MONGO_PORT for port communication